emcc.py

```diff
@@ -1577,7 +1577,7 @@ def is_supported_link_flag(f):
       shared.Settings.AUTO_JS_LIBRARIES = 0
 
       # In asm.js always use memory init file to get the best code size, other modes are not currently supported.
-      if not shared.Settings.WASM:
+      if not shared.Settings.WASM and not shared.Settings.WASM_BACKEND:
```

Comment:
I think we still use a mem init file for wasm2js (wasm=0 with wasm backend). But maybe that's not handled in this code?

Reply:
Yeah, this line only specifies memory init file for fastcomp asm.js builds.